jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/impl/module/SimpleTypeModule.java

```diff
@@ -82,10 +82,12 @@ public static SimpleTypeModule forPrimitiveAndAdditionalTypes() {
         SimpleTypeModule module = SimpleTypeModule.forPrimitiveTypes();
 
         module.withStringType(java.time.LocalDate.class, "date");
-        Stream.of(java.time.LocalDateTime.class, java.time.LocalTime.class, java.time.ZonedDateTime.class,
-                java.time.OffsetDateTime.class, java.time.OffsetTime.class, java.time.Instant.class,
+        Stream.of(java.time.LocalDateTime.class, java.time.ZonedDateTime.class,
+                java.time.OffsetDateTime.class, java.time.Instant.class,
                 java.util.Date.class, java.util.Calendar.class)
                 .forEach(javaType -> module.withStringType(javaType, "date-time"));
+        Stream.of(java.time.LocalTime.class,  java.time.OffsetTime.class)
```

Comment:
_There shall be only one!_ 😉
```suggestion
        Stream.of(java.time.LocalTime.class, java.time.OffsetTime.class)
```